daras_ai_v2/base.py

```diff
@@ -2276,43 +2280,255 @@ def render_published_run_preview(self, published_run: PublishedRun):
+                    ):
+                        with gui.div(
+                            className="d-flex",
+                            style={"font-size": "0.7rem", "marginLeft": "46px"},
+                        ):
+                            for pill in pills:
+                                pill()
+                    with gui.div(className="d-flex gap-1 align-items-center"):
+                        (
+                            self.render_author(
+                                published_run.last_edited_by,
+                                image_size="14px",
+                                text_size="0.8rem",
+                                responsive=False,
+                                show_as_link=False,
+                            )
+                            if not use_workspace_author
+                            else self.render_workspace_author(
+                                published_run.workspace,
+                                image_size="14px",
+                                text_size="0.8rem",
+                                responsive=False,
+                                show_as_link=False,
+                            )
+                        )
+                        if updated_at and isinstance(updated_at, datetime.datetime):
+                            gui.caption(
+                                f" • {get_relative_time(updated_at)}",
+                                style={"fontSize": "0.8rem"},
+                                className="container-margin-reset",
+                            )
+
+                # web
+                with gui.div(
+                    className="d-none d-md-block text-end white-space-nowrap container-margin-reset",
+                    style={"whiteSpace": "nowrap"},
+                ):
+                    if hide_author_column:
+                        with gui.div(className="mb-2"):
+                            (
+                                self.render_author(
+                                    published_run.last_edited_by,
+                                    image_size="24px",
+                                    text_size="0.9rem",
+                                    responsive=False,
+                                    show_as_link=False,
+                                    className="mb-2",
+                                )
+                                if not use_workspace_author
+                                else self.render_workspace_author(
+                                    published_run.workspace,
+                                    image_size="24px",
+                                    text_size="0.9rem",
+                                    responsive=False,
+                                    show_as_link=False,
+                                )
+                            )
```

Comment:
This would be easier to read with proper `if-else` conditions. 

```python
if not use_workspace_author:
   self.render_author(...)
else:
   self.render_workspace_author(...)
```

Please use the same elsewhere when it spans multiple lines